core/platforms/freebsd/bareos-freebsd/bareos.com-bconsole/Makefile

```diff
@@ -7,5 +7,7 @@ MASTERDIR=	${.CURDIR}/../bareos.com-common
 
 LIB_DEPENDS+=  libbareos.so:sysutils/baroes.com-common
 
+# set: WRKDIR =
+
```

Comment:
I guess these are used as markers for script-based editing, right.
Maybe it is worth to add a comment to each of these, so they aren't accidentially removed in the future.

Reply:
Wouldn't it be easier if the script could just create a file in MASTERDIR that gets included if it exists.
i.e. something like
```
.-include $(MASTERDIR)/overrides.mk
```
which will include `overrides.mk` if it exists and do nothing otherwise.

Reply:
applied.